components/MonthSelection.js

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react'
+
+const MonthSelection = ({onMonthChange, onYearChange}) => {
```

Comment:
```suggestion
const MonthSelection = ({onMonthChange, onYearChange, month, year}) => {
```

The year and month should be received as properties